pkg/webhook/admission/defaulter_custom.go

```diff
@@ -0,0 +1,85 @@
+/*
+Copyright 2018 The Kubernetes Authors.
```

Comment:
```suggestion
Copyright 2021 The Kubernetes Authors.
```
nit (same in validator_custom.go)